force-app/main/default/lwc/systemTools/systemTools.js

```diff
@@ -82,6 +98,16 @@ export default class systemTools extends LightningElement {
         return this.labelReference.stgRunCleanUpEnableFirstTime + " " + this.prefEmailPhoneHyperLink;
     }
 
+    get ethnicityAndBackFillHyperLink() {
+        return (
+            '<a href="https://powerofus.force.com/s/article/EDA-Contact">' + this.labelReference.tellMeMoreLink + "</a>"
```

Comment:
Since the target is a constant, we might consider making it a constant outside the module itself. This is not necessary, but it would make updating it easier.

Reply:
will update all the links in the next batch PR.